invenio_orcid/tasks.py

```diff
@@ -152,12 +170,11 @@ def get_author_collection_records_from_valid_authors(authors_refs):
             }
         }
     }
-    authors = current_search_client.search(
-        index='records-authors',
-        doc_type='authors',
-        body=es_query
-    )['hits']['hits']
-    return authors
+
+    AuthorSearch = import_string(
```

Comment:
It's good idea to move it to the state object of InvenioORCID extension.

--
Check the pattern in https://github.com/inveniosoftware/invenio-access/blob/master/invenio_access/ext.py